bpf_test.go

```diff
@@ -253,11 +261,105 @@ func checkPinConfigCleanup(t *testing.T, expectedPaths []string) {
 	}
 }
 
+func checkUpdateDeleteElement(t *testing.T, b *elf.Module) {
+	mp := b.Map("dummy_hash")
+	if mp == nil {
+		t.Fatal("unable to find dummy_hash map")
+	}
+
+	key := 1000
+	value := 1000
+	if err := b.UpdateElement(mp, unsafe.Pointer(&key), unsafe.Pointer(&value), BPF_ANY); err != nil {
+		t.Fatal(err)
```

Comment:
Can you add some context to these errors? Something like:

```
t.Fatal("Failed trying to update an element with BPF_ANY")
```

And in the next one:

```
t.Fatal("Succeeded updating element with BPF_NOEXIST, but an element with the same key was added to the map before") 
```

and so on.